notes/condition.md

```diff
@@ -12,27 +12,22 @@ sort: 11
 *   Compute the condition number
 *   Quantify the impact of a high condition number
 
-## Condition Number Definition
+## Numerical experiments
 
-The **_condition number_** of a square nonsingular matrix <span>$${\bf A}$$</span> is defined by
-$$\text{cond}({\bf A}) = \kappa({\bf A}) = \|{\bf A}\| \|{\bf A}^{-1}\| $$
-which is also the condition number associated with solving the linear system $${\bf A} \boldsymbol{x} = \boldsymbol{b}$$. A matrix with a large condition number is said to be **_ill-conditioned_**.
+**Input** has uncertainties:
 
-The condition number can be measured with any <span>$$p$$</span>-norm, so to be precise we typically specify the norm being used, i.e. $$\text{cond}_2$$, $$\text{cond}_1, \text{cond}_{\infty}$$.
+* Errors due to representation with finite precision
+* Error in the sampling
 
-If <span>$${\bf A}$$</span> is singular ($${\bf A}^{-1}$$ does not exist), we can define $$\text{cond}({\bf A}) = \infty$$ by convention.
+Once you select your numerical method, how much error should you expect to see in your output?
 
-The identity matrix is well conditioned. Assuming the inverse of $$\|{\bf A}\|$$ exists, $$\text{cond}({\bf A}) = \|{\bf A}\| \|{\bf A}^{-1}\| \geq \|{\bf A}{\bf A}^{-1}\| = \|{\bf I}\| = 1.$$ This is the smallest possible condition number.
+_Is your method sensitive to errors (perturbation) in the input?_
 
-#### Things to Remember About Condition Numbers
-*   For any matrix $${\bf A}$$, $$\text{cond}({\bf A}) \geq 1.$$
-*   For the identity matrix $${\bf I}$$, $$\text{cond}({\bf I}) = 1.$$
-*   For any matrix $${\bf A}$$ and a nonzero scalar $$\gamma$$, $$\text{cond}(\gamma {\bf A}) = \text{cond}({\bf A}).$$
-*   For any diagonal matrix $${\bf D}$$, $$\text{cond}({\bf D}) $$ = $$\frac{\text{max}\mid d_{i} \mid}{\text{min}\mid d_{i} \mid}.$$
-*   The condition number is a measure of how close a matrix is to being singular: a matrix with large condition number is nearly singular, whereas a matrix with a condition number close to 1 is far from being singular.
-*   The determinant of a matrix is **NOT** a good indicator to check whether a matrix is near singularity.
 
-## Perturbed Matrix Problem and Error Bound
+## Sensitivity of Solutions of Linear Systems and Error Bound
+
+Suppose we start with a non-singular system of linear equations $${\bf A} \boldsymbol{x} = \boldsymbol{b}$$. We change the right-hand side vector $$\boldsymbol{b}$$ (the input) by a small amount $$\Delta \boldsymbol{b}$$. How much does the solution $$\boldsymbol{x}$$ (the output) change, i.e., how large is $$\Delta \boldsymbol{x}$$. 
+
 
 Let $$\boldsymbol{x}$$ be the solution of $${\bf A} \boldsymbol{x} = \boldsymbol{b}$$ and $$\hat{\boldsymbol{x}}$$ be the solution of the perturbed problem $${\bf A} \hat{\boldsymbol{x}} = \boldsymbol{b} + \Delta \boldsymbol{b}$$. Let $$\Delta \boldsymbol{x} = \hat{\boldsymbol{x}} - \boldsymbol{x}$$ be the absolute error in output. Then we have
```

Comment:
Format so everything isn't on one line. Transitions, etc to make things flow better.